schedule/yast/msdos.yaml

```diff
@@ -0,0 +1,73 @@
+---
+name:           msdos@yast
+description:    >
+  Test for installation on msdos partition table.
+schedule:
+  - installation/bootloader_start
+  - installation/welcome
+  - installation/accept_license
+  - installation/scc_registration
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning/msdos_partition_table
+  - installation/installer_timezone
+  - installation/user_settings
+  - installation/user_settings_root
+  - installation/resolve_dependency_issues
+  - installation/installation_overview
+  - {{disable_grub_timeout}}
+  - installation/first_boot
+  - console/validate_fs_table
+conditional_schedule:
+  disable_grub_timeout:
+    MACHINE:
+      64bit:
+        - installation/disable_grub_timeout
+        - installation/start_install
+        - installation/await_install
+        - installation/logs_from_installation_system
+        - installation/reboot_after_installation
+        - installation/grub_test
+      ppc64le:
+        - installation/start_install
+        - installation/await_install
+        - installation/logs_from_installation_system
+        - installation/reboot_after_installation
+      ppc64le-hmc-single-disk:
+        - installation/edit_optional_kernel_cmd_parameters
+        - installation/disable_grub_timeout
+        - installation/start_install
+        - installation/await_install
+        - installation/logs_from_installation_system
+        - installation/reboot_after_installation
+        - boot/reconnect_mgmt_console
+        - installation/grub_test
+      s390x-kvm-sle12:
+        - installation/disable_grub_timeout
+        - installation/start_install
+        - installation/await_install
+        - installation/logs_from_installation_system
+        - installation/reboot_after_installation
+        - boot/reconnect_mgmt_console
+      svirt-xen-hvm:
+        - installation/disable_grub_timeout
+        - installation/start_install
+        - installation/await_install
+        - installation/logs_from_installation_system
+        - installation/reboot_after_installation
+        - installation/grub_test
+      svirt-xen-pv:
+        - installation/start_install
+        - installation/await_install
+        - installation/logs_from_installation_system
+        - installation/reboot_after_installation
```

Comment:
These 4 modules are same for all, so I would propose to create 2 conditions, one for the test modules before these 4 and one for the test modules after.

Reply:
logs_from_installation_system do not work for hyperv

Reply:
That's right, but you still have 3 modules as common denominator:
```
- installation/start_install
- installation/await_install
- installation/reboot_after_installation
```

Reply:
I had added 3 conditions, instead of one, but it looked a bit confusing. Attached
[Attached](https://github.com/os-autoinst/os-autoinst-distri-opensuse/files/4213318/msdos.txt)
 is what initial schedule looks like. If you think is better this way, I can change it back.

Reply:
Thanks for taking a look again, in fact it looks more clear in your attachment because then we know better for each test module what is going on.
Please double check current schedule, because I spot for example `disable_grub_timeout` in ppc64le and I think it should be there like in your attachment.
That way it is in fact the original idea of the how to do the conditional schedule, the problem is that we never came up with a syntaxis for exclusion or way to assign one test module to a list of possible values in the yaml schedule parser. So I would go for the way in the attachment and if hurts too much our eyes, we can always to modify the parser. But once you know that, it is up to you if you still want to keep this 'block' way (I would call it like that due it targets specific modules) as it is fine too (just fix that omission and double-check)

Reply:
Ok, I'll double check and rerun tests after changes